core/src/main/scala/stryker4s/mutants/findmutants/MutantMatcher.scala

```diff
@@ -84,12 +86,14 @@ class MutantMatcher()(implicit config: Config) {
         mutationToTerm: T => Term
     ): Seq[Option[Mutant]] =
       ifNotInAnnotation {
-        mutations map { mutated =>
-          if (matchExcluded(mutated))
-            None
-          else
-            Some(Mutant(ids.next, original, mutationToTerm(mutated), mutated))
-        }
+        mutations
+          .filterNot(isSuppressedByAnnotation(_, original))
```

Comment:
Can you move this to the `if` in the map-statement so it will count as excluded? That will probably make it easier in the future to show this as `Ignored` status in the report.

Reply:
Thanks for review! 100% agree with all your notes. Just pushed changes.